roles/kubernetes-apps/metallb/templates/metallb.yml.j2

```diff
@@ -339,7 +339,7 @@ spec:
           readOnlyRootFilesystem: true
       hostNetwork: true
       nodeSelector:
-        beta.kubernetes.io/os: linux
+        {{ 'beta.' if kube_version is version('v1.18.0', '<') }}kubernetes.io/os: linux
```

Comment:
Why bother with the if ? This is going to be merged with kubespray that deploy k8s 1.19

Reply:
Nope.
Kubespray may install all supported releases. from 1.17.0 to last added for now.

Reply:
I thought it was just 1 before.